install.sh

```diff
@@ -409,6 +437,8 @@ install_get_os ()
 {
     if os_is_deb; then
         echo 'debian'
+    elif os_is_dar; then
+	echo 'darwin'
```

Comment:
Indent this please.

Reply:
Umm do you mean have it be 4 more spaces to the right? I thought the elif should be aligned with the if at line 438.

Reply:
Copy paste a portion or all of the shell script into a formatter for a 2nd opinion, e.g.: https://www.shellcheck.net/

Reply:
no issues detected with current style